### DISCUSSION TITLE
[Feature] Save/Bookmark coordinates of locations for GPS editing

### DISCUSSION THREAD

**Discussion by radh21301** [immich-app/immich#12378] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A big portion of my images are taken in a few select locations. Say:
* Home
* Work
* Kid's school
* Parent's home
* ...

So, it would be easier if we can bookmark/save the GPS coordinates of these locations to help future editing instant. All you would need to do is, hit edit location, select 'Home', and done. Much much faster compared to searching for a place in the GUI, or getting the GPS coordinates from elsewhere and pasting it in Immich etc.

P.S. Digikam has this feature and it is extremely handy.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9191

**Comment by GikikCrowchew** [comment]
I agree, an addressbook with corresponding gps coorinates and the ammount of times these coordinates where linked with picture.
Then when you have a picture without coordinates you can eather use the current system or just quick-pick from the list of already known coordinates. A lot of pictures are at known locations like, home, family member home, ... and now its more work.
if this is too hard having the coordinates in a single copy past field could speed this up as well

**Comment by mylbeee** [comment]
This would indeed be a great feature to have! Searching the map is too hard for each photo, and the locations won't be exactly the same between photos. Copy-paste of lat/long values is not really an option.

I know you can geotag multiple pictures at once, but in my work flow I'd rather tag the one by one when editing the date and people tags.
